src/manifest.json

```diff
@@ -0,0 +1,55 @@
+{
+   "app_id": "drive",
```

Comment:
I'm hesitant to "burn" a generic app ID as `drive`. Maybe use a more specific one like `google-drive`?